thorvg/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "0.13.8"
+version: "0.13.8~1"
```

Comment:
@espressif2022 how about upgrade the library to sync with the upstream version v0.14.6?

Reply:
> @espressif2022 how about upgrade the library to sync with the upstream version v0.14.6?

Last week I encountered some memory loading file loading failures. I tried the new version, and it seems that the new version will encounter some `sys/mmap.h` compilation problems. I will confirm it again.

Reply:
> > @espressif2022 how about upgrade the library to sync with the upstream version v0.14.6?
> 
> Last week I encountered some memory loading file loading failures. I tried the new version, and it seems that the new version will encounter some `sys/mmap.h` compilation problems. I will confirm it again.

Should enable -o1 option, 0.14.6 works.

Reply:
That looks like a workaround... What is the error you were getting?

Reply:
> That looks like a workaround... What is the error you were getting?

I mainly want to add the option of `-Db_staticpic=false`. As mentioned below, some lottie playbacks will crash (when testing lottie files from customers).
Because this option needs to be added, other options are also added to kconfig at the same time.